src/Ninja/G1CP/Content/Tests/test003.d

```diff
@@ -6,9 +6,15 @@
  * Expected behavior: The NPC will pick up and equip the sword, leave the bow, draw the sword and show it
  */
 func void G1CP_Test_003() {
-    if (!G1CP_TestsuiteAllowManual) {
-        return;
-    };
+    G1CP_Testsuite_CheckManual();
+    G1CP_Testsuite_CheckAiStateStartFunc("ZS_PracticeSword");
+    var int symbId1; symbId1 = G1CP_Testsuite_CheckItem("Whistlers_Schwert");
+    var int symbId2; symbId2 = G1CP_Testsuite_CheckItem("Wolfs_Bogen");
+    var int symbId3; symbId3 = G1CP_Testsuite_CheckItem("ItAmArrow");
```

Comment:
To add readability, I would rename the variables to `swordId`, `bowId` and `arrowId` respectively.

Reply:
But don't we loose copy&paste-bility with this again?

Reply:
With the numbers behind `symbId` you'd have to rename the variable anyway.